ConsoleApp/School/Schedule.cs

```diff
@@ -0,0 +1,30 @@
+namespace School;
+
+class Schedule
+{
+    public string[][] Lessons {get;set;}
+    public string[] GetScheduleForDay(string day) // DateTime.Now.DayOfWeek.ToString()
```

Comment:
there is enum DayOfWeek in .NET you can use instead
https://docs.microsoft.com/en-us/dotnet/api/system.dayofweek?view=net-6.0